mne/viz/ica.py

```diff
@@ -146,8 +146,8 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
     # we create a new epoch with dropped rows
     epoch_data = epochs_src.get_data()
     epoch_data = np.insert(epoch_data,
-                           dropped_indices -
```

Comment:
I would rather see why dropped_indices is not an int as it's weird for me that an array called indices is not of dtype int